ev11l78a: minimize memory usage in the default config.

Flameeyes opened the pull request:
ev11l78a: minimize memory usage in the default config.

The ATSAMD20E16 used in this board only has 8KiB SRAM, so the defaults
are not really suitable for anything close to useful, particularly the
sink sample that most closely resembles the original demo for the board.

This should also allow more tests not to be skipped for this board as
otherwise they overflow RAM usage.

Signed-off-by: Diego Elio Pettenò <flameeyes@meta.com>

---
Stack created with [Sapling](https://sapling-scm.com). Best reviewed with [ReviewStack](https://reviewstack.dev/zephyrproject-rtos/zephyr/pull/61797).
* #61740
* #62478
* __->__ #61797

nandojve commented on boards/arm/adafruit_trinket_m0/adafruit_trinket_m0.dts:
```diff
@@ -6,6 +6,7 @@
 
 /dts-v1/;
 #include <atmel/samd21.dtsi>
+#include <atmel/samd2xx18.dtsi>
```
This should follow same solution of #61744 which is include main header inside `samd2xx18.dtsi`

nandojve commented:
You need rebase : )

nandojve commented:
Ping

Flameeyes committed:
ev11l78a: minimize memory usage in the default config.

The ATSAMD20E16 used in this board only has 8KiB SRAM, so the defaults
are not really suitable for anything close to useful, particularly the
sink sample that most closely resembles the original demo for the board.

This should also allow more tests not to be skipped for this board as
otherwise they overflow RAM usage.

Signed-off-by: Diego Elio Pettenò <flameeyes@meta.com>

Flameeyes commented:
Sorry, been a busy few months and this project took a significant back burner status. But rebased this PR as I think nothing stops it.
